src/sdk/pynni/nni/nas/pytorch/search_space_zoo/darts_search_space.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import torch.nn as nn
+import ops
+
+
+class DartsStackedCells(nn.Module):
+    """
+    builtin Darts Search Space
+    Compared to Darts example, DartsSearchSpace removes Auxiliary Head, which
+    is considered as a trick rather than part of model.
+
+    Attributes
+    ---
+    in_channels: int
+        the number of input channels
+    channels: int
+        the number of initial channels expected
+    n_classes: int
+        classes for final classification
+    n_layers: int
+        the number of cells contained in this network
+    factory_func: function
+        return an instance for certain cell structure
+        user should pass in `__init__` of demand cell class
```

Comment:
Should write
```
``__init__``
```

Suggest to make the signature clearer here. For example, it can be a factory function with the following parameters, and return an object callable with the following parameters...

Reply:
I listed some required parameters. But I believe the best way is referring to an example.